aspnetcore/data/ef-mvc/migrations.md

```diff
@@ -165,21 +167,10 @@ For more information about the PMC commands, see [Package Manager Console (Visua
 
 ## Get the code
 
-[Download or view the completed application.](https://github.com/dotnet/AspNetCore.Docs/tree/master/aspnetcore/data/ef-mvc/intro/samples/cu-final)
+[Download or view the completed application.](https://github.com/dotnet/AspNetCore.Docs/tree/master/aspnetcore/data/ef-mvc/intro/samples)
```

Comment:
I would continue to send them to cu-final.  In /samples there are four to choose from and people won't have a clue which one to choose.